back-office/advanced-parameters/database/advanced_parameters-database-sql_manager.md

```diff
@@ -1,8 +1,59 @@
 # SPECIFICATIONS - ADVANCED PARAMETERS &gt; DATABASE &gt; SQL MANAGER
 
-## Grid
+The SQL manager is a complex feature, which should be reserved for technical people who know how to explore a database using the SQL language. In return for this complexity, it can be extremely powerful and proves immensely useful to those who master it.
 
-Default action in the button: Export Other available actions in the menu: View, Edit, Delete Row action: View
+## 1.7.8 Version 
+The page is going to be migrated into a module. The module will be downloadable for merchants who need it.
+The feature to hide the password or the secure key will be removed since we can't fully hide them since with an alias the keys or passwords can be displayed.
```

Comment:
Are you talking of that ?

![capture d'écran_2829](https://user-images.githubusercontent.com/13449658/96855592-3ba33d80-145d-11eb-8428-d165fdcd5f2a.png)

Reply:
Yes, @PierreRambaud was saying that it can be easily bypassed.

Reply:
All right, while reading I thougt you were talking of a feature like a "Show / hide password" button :smile: 
Maybe it will be more clearer if you add a screenshot ? Or say directly that they will be displayed in clear?